config/deploy.rb

```diff
@@ -58,6 +59,7 @@
     end
   end
 
+  #TODO FIX: remove password from wp core install command
   desc "Wp Core Install"
```

Comment:
Ok, that makes sense. But yes, I still think it makes more sense as part of a WordPress role in playbook.